projects/baby_driver/scripts/i2c_write.py

```diff
@@ -0,0 +1,50 @@
+"""Python implementation of the i2c_write function"""
+import can_util
+from message_defs import BabydriverMessageId
+
+
+def i2c_write(port,address, tx_bytes, reg=None):
+    """
+    Writes over I2C to the given port/address
+    Args:
+        port: port of the I2C to write to (1 or 2)
+        address: I2C address to write to (0-255)
+        tx_bytes: list of bytes to write over I2C (a list of ints 0-255)
+        reg: the register to write to, or None if no register is required
+    Raises:
+        Exception: if a non-zero status code was received
+    """
+    if port not in (1,0):
+        raise ValueError("Expected port of 0 (I2C_PORT_1) or 1 (I2C_PORT_2)")
+    if address <0 or address > 255:
+        raise ValueError("Expected address between 0 and 255")
+    # pylint: disable=consider-using-enumerate
+    for i in range(len(tx_bytes)):
+        if tx_bytes[i] <0 or tx_bytes[i] > 255:
+            raise ValueError("Expected list of bytes between 0 and 255")
```

Comment:
Ah, I thought pylint was talking about your `range(0, len(tx_bytes), 7)` construction below with its consider-using-enumerate warning. It would be more idiomatic ("pythonic") to iterate directly over the list with `for byte in tx_bytes`, since you're not using the index beyond indexing into the list, but I think this might be the most pythonic way to do this condition:
```py
if not all(0 <= byte < 256 for byte in tx_bytes):
  raise ValueError("Expected list of bytes between 0 and 255")
```
This works! The thing in the brackets is a [generator comprehension](https://www.pythonlikeyoumeanit.com/Module2_EssentialsOfPython/Generators_and_Comprehensions.html) (like a list comprehension if you've seen those, but without brackets and not computing elements until they're asked for), `all` is a function which returns True if and only if all the values of its argument are True, and `0 <= byte < 256` is equivalent to `0 <= byte and byte < 256` because Python is magic.

This way you can remove the `# pylint: disable` comment.